src/ol_concourse/lib/resources.py

```diff
@@ -91,14 +91,18 @@ def github_issues(  # noqa: PLR0913
     name: Identifier,
     repository: str,
     issue_prefix: str,
+    auth_method: Literal["token", "app"] = "token",
     access_token: str = "((github.issues_resource_access_token))",  # noqa: S107
+    app_id: str = "((github.issues_resource_app_id))",
+    app_installation_id: str = "((github.issues_resource_app_installation_id))",
+    private_ssh_key: str = "((github.issues_resource_private_ssh_key))",
```

Comment:
Where is this value coming from?

Reply:
Like where did the key come from? Github generates it downloads it when you setup an application. It is being stored into vault else where in this PR.